dbt_project.yml

```diff
@@ -30,6 +30,8 @@ models:
           +materialized: view
         intermediate:
           +materialized: ephemeral
+          int_ef3__deduped_descriptors:
+           +materialized: table
```

Comment:
I've discovered a bug in RLS-enabled implementations. Because this model lacks a "tenant_code" field, it fails at compile.

To resolve this, add the "bypass_rls" tag to this model in dbt_project.yml.